### PR TITLE
cmd:  add cli to generate  service accounts

### LINKED ISSUE #492: Programmatic access with Google service_account
**Is your feature request related to a problem? Please describe.**

We want to implement automated end-to-end tests for our pomerium-secured web applications, which involves a Google account. However, we do not really want to use an actual user account but rather a service account.

By following the instructions on [Programmatic access](https://www.pomerium.io/docs/reference/programmatic-access.html), I could not find it possible to use a service account to do so.

**Describe the solution you'd like**

Make it possible for service account to be authenticated via Pomerium without human intervention.

### PR DESCRIPTION
**Pull request by desimone**
## Summary

This adds a command line interface to generates pomerium service accounts using the same shared key as your proxy service. 

NOTA BENE: These personas will be valid until the expiry (defaults to one hour) or the shared key is changed. 

## Related issues

Fixes #492 

## Pictures 

Screen Shot 2020-03-21 at 11 44 53 AM

**Checklist**:
- [x] add related issues
- [ ] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] ready for review

### CODE REVIEW

**Comment by desimone** [comment]
\cc @travisgroth if there's an easy way to make this work with Cobra that isn't 5x the LOC I haven't figured it out. If you are cool with it, I'd like to merge this (assuming passes review) as is and refactor with cli / cobra later.

File: cmd/pomerium-cli/cli.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+		return errors.New("iss is required")
+	}
+
+	// check that we've got our shared key to sign our jwt
+	args := flags.Args()
+	if len(args) != 1 {
+		return errors.New("shared key required")
+	}
+
+	sharedKey := args[0]
```
Comment (travisgroth) [inline cmd/pomerium-cli/cli.go]: I feel a little odd asking for this key as a command line arg without another option.  Stdin and/or a file path containing the secret would be good alternatives less prone to leaking in shell history, etc.

**Comment by desimone** [comment]
updated !
carbon

File: cmd/pomerium-cli/cli.go (hunk 1 above)
Comment (desimone) [reply cmd/pomerium-cli/cli.go]: Updated! Still allows insecure-y argument but has option for stdin input.

carbon

Comment (desimone) [reply cmd/pomerium-cli/cli.go]: \cc @freeyoung how's this look?

Comment (freeyoung) [reply cmd/pomerium-cli/cli.go]: lgtm 👍

**Pull request merged**